doc/man1/openssl-s_server.pod.in

```diff
@@ -692,6 +646,12 @@ by the client in binary mode.
 
 {- $OpenSSL::safe::opt_engine_item -}
 
+{- $OpenSSL::safe::opt_v_item -}
+
+If the server requests a client certificate, than
+verification errors are displayed, for debugging, but the command will
+proceed unless the B<-verify_return_error> option is used.
+
```

Comment:
Is it a real behavior when the client certificate is requested?

Reply:
Yes, look at the verify_callback function in apps/lib/s_cb.c